restapi/api/routes.go

```diff
@@ -8,6 +8,8 @@ func registerRoutes(router *gin.RouterGroup) {
 	device.Use(DeviceMiddleware())
 	device.GET("/info", Info)
 	device.GET("/screenshot", Screenshot)
+	device.POST("/setlocation", SetLocation)
```

Comment:
set location is technically a `PUT` operation, right?

Reply:
Mmm I am not really sure `PUT` is applicable in this case because this is not a resource we are modifying, but either way is acceptable for me, let me know if you prefer to use `PUT`

Reply:
yes because PUT is an idempotent operation. Meaning, if I call it n-times with the same parameters, there will be no change in system state, which is the case here.
It does not matter if I call setlocation?lat=90&lon=90 once, twice or 500 times, the result will be the same.

Reply:
(idempotency is a really nice property to have in distributed systems, so the endpoint should be proud of it)

Reply:
that makes sense, done